include/level_object_entries.h

```diff
@@ -0,0 +1,373 @@
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_LavaSpurt;
+
+typedef struct LevelObjectEntry_PosArrow {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_PosArrow;
+
+typedef struct LevelObjectEntry_HitTester {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_HitTester;
+
+typedef struct LevelObjectEntry_MidiFade {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_MidiFade;
+
+typedef struct LevelObjectEntry_DynamicLightingObject {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_DynamicLightingObject;
+
+typedef struct LevelObjectEntry_EffectBox {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_EffectBox;
+
+typedef struct LevelObjectEntry_TrophyCab {
+    /* 0x00 */ LevelObjectEntryCommon common;
+    /* 0x08 */ u8 rotation;
+} LevelObjectEntry_TrophyCab;
+
+typedef struct LevelObjectEntry_Bubbler {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_Bubbler;
+
+typedef struct LevelObjectEntry_FlyCoin {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_FlyCoin;
+
+typedef struct LevelObjectEntry_GoldenBalloon {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_GoldenBalloon;
+
+typedef struct LevelObjectEntry_Laserbolt {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_Laserbolt;
+
+typedef struct LevelObjectEntry_Lasergun {
+    /* 0x00 */ LevelObjectEntryCommon common;
+    /* 0x08 */ u8 unk8;
+    /* 0x09 */ u8 unk9;
+    /* 0x0A */ s8 unkA;
+    /* 0x0B */ s8 unkB;
+    /* 0x0C */ u8 unkC;
+    /* 0x0D */ u8 unkD;
+} LevelObjectEntry_Lasergun;
+
+typedef struct LevelObjectEntry_GroundZipper {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_GroundZipper;
+
+typedef struct LevelObjectEntry_OverRidePos {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_OverRidePos;
+
+typedef struct LevelObjectEntry_WizpigShip {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_WizpigShip;
+
+typedef struct LevelObjectEntry_SilverCoin {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_SilverCoin;
+
+typedef struct LevelObjectEntry_Boost {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_Boost;
+
+typedef struct LevelObjectEntry_WardenSmoke {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_WardenSmoke;
+
+typedef struct LevelObjectEntry_Unknown94 {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_Unknown94;
+
+typedef struct LevelObjectEntry_Unknown96 {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_Unknown96;
+
+typedef struct LevelObjectEntry_Snowball {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_Snowball;
+
+typedef struct LevelObjectEntry_Teleport {
+    /* 0x00 */ LevelObjectEntryCommon common;
+} LevelObjectEntry_Teleport;
+
+typedef struct LevelObjectEntry_Lighthouse_RocketSignpost {
```

Comment:
```suggestion
typedef struct LevelObjectEntry_Lighthouse {
```

Reply:
I wrote it like this to denote that this entry is both for the lighthouse and the beach signpost objects. It would be a little confusing to the reader if the name only had one. I do agree that having two underscores does look a little bad. Do you think  `LevelObjectEntry_LighthouseRocketSignpost` would be fine? Or would another name be better?

Reply:
I see, in that please change the name of the typedef (it's currently `LevelObjectEntry_Lighthouse`).

I prefer two underscores. `LevelObjectEntry_LighthouseRocketSignpost` makes me think that `LighthouseRocketSignpost` is a single thing.